src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue27935.cs

```diff
@@ -0,0 +1,22 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue27935 : _IssuesUITest
+	{
+		public Issue27935(TestDevice device) : base(device) { }
+
+		public override string Issue => "iOS: Rectangle that is invisible when page loads can never be made visible";
+
+		[Test]
+		[Category(UITestCategories.Shape)]
+		public void ShapesShouldAppearCorrectlyWhenIsVisibleChanges()
+		{
+			App.WaitForElement("Button");
+			App.Tap("Button");
+			VerifyScreenshot();
```

Comment:
Can take two screenshots to verify the two possible IsVisible property states? (also when the shapes are hidden)